code/modules/migrations/SS13_Prefs/007-remove-flavortext.dm

```diff
@@ -1,14 +1,13 @@
-/datum/migration/sqlite/ss13_prefs/_007
-	id = 7
-	name = "Remove flavor text"
+/datum/migration/sqlite/ss13_prefs/_009
```

Comment:
You don't need to change this file at all, just revert it back to what it originally was.